mirdata/download_utils.py

```diff
@@ -129,19 +117,11 @@ def download_from_remote(remote, save_dir, force_overwrite=False):
     if not os.path.exists(download_path) or force_overwrite:
         # If file doesn't exist or we want to overwrite, download it
         with DownloadProgressBar(
-            unit='B', unit_scale=True, miniters=1, desc=remote.url.split('/')[-1]
+            unit='B', unit_scale=True, unit_divisor=1024, miniters=1
         ) as t:
-            try:
-                _download_large_file(remote.url, download_path, t.update_to)
-            except HTTPError:
-                error_msg = """
```

Comment:
What will happen now if there's a dead link? Is there a way to catch that specific error explicitly and keep the error message below?

Reply:
Updated so we will raise the error with the same message as before.